plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -956,8 +958,10 @@ public void testNullPredicates()
     }
 
     @Test
+    @Override
     public void testLimitPushdown()
     {
+        super.testLimitPushdown();
         assertThat(query("SELECT name FROM nation LIMIT 30")).isFullyPushedDown(); // Use high limit for result determinism
```

Comment:
The base `testLimitPushdown` method has the same assertion.

Reply:
removed